lib/screens/start_screen.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+import 'package:provenance_wallet/screens/home/home_screen.dart';
+import 'package:provenance_wallet/screens/landing/landing_screen.dart';
+import 'package:provenance_wallet/util/get.dart';
+import 'package:provenance_wallet/util/local_auth_helper.dart';
+
+class StartScreen extends StatefulWidget {
+  const StartScreen({Key? key}) : super(key: key);
+
+  @override
+  State<StatefulWidget> createState() => StartScreenState();
+}
+
+class StartScreenState extends State<StartScreen> {
+  late LocalAuthHelper _localAuthHelper;
+
+  @override
+  void initState() {
+    super.initState();
+    _localAuthHelper = get<LocalAuthHelper>();
+    _localAuthHelper.auth(context);
```

Comment:
So, does this still do the auto-navigation to the dashboard?

Reply:
no. start_screen will listen to the status stream and route accordingly. Nothing should need to push the landing/dashboard

Reply:
sorry, misread the question. yes, the stream builder will auto navigate based on the status value